### PR TITLE
docs: add commands about testing, formatting, and linting

### LINKED ISSUE #851: Update CII badge after adding data validation tests
As [mentioned here](https://github.com/MetabolicAtlas/MetabolicAtlas/issues/780#issuecomment-1014258352), the badge should be updated once #830 is merged.

### PR DESCRIPTION
**Pull request by e0**
**Related issue(s) and PR(s)**  
This PR closes #851.

**Type of change**  

- [x] Other
 
**List of changes made**  

Add commands about testing, formatting, and linting to README.

**Checklist**  

- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,11 @@ deploy-stack dev
 * To bootstrap the project: `build-stack`
 * To run the project: `start-stack`
 * To stop the project: `stop-stack`
+* To run a command inside a docker container: `ma-exec <CONTAINER_NAME> <COMMAND>`. Below is a list of useful container specific commands.
+  * Run API data validation tests: `ma-exec api yarn test`
+  * Use prettier to format frontend (also works for `api`) files: `ma-exec frontend format`
```
Comment (nanjiangshu) [inline README.md:82]: Should be `ma-exec frontend yarn format`

File: README.md [hunk 2]
```diff
@@ -77,6 +77,11 @@ deploy-stack dev
...
+  * Run eslint for frontend (also works for `api`): `ma-exec frontend lint`
```
Comment (nanjiangshu) [inline README.md:83]: Should be `ma-exec frontend yarn lint`

Comment (nanjiangshu) [reply README.md]: A bit unclear what's the different between `yarn lint` and `yarn format`. Maybe briefly describe it or link to further reading?

Comment (e0) [reply README.md]: Thank you for spotting these @nanjiangshu! Fixed now.

**Review by nanjiangshu — COMMENTED** [review]
Great that you added instructions for these commands @e0. I just added a few minor comments.

File: README.md (hunk 2 above)
Comment (e0) [reply README.md]: > A bit unclear what's the different between `yarn lint` and `yarn format`. Maybe briefly describe it or link to further reading?

We have some more information about lint and prettier [here](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/CONTRIBUTING.md#prettier). I would prefer to keep this section (list of commands) as light as possible if everyone is ok with that.

Comment (nanjiangshu) [reply README.md]: It makes sense to keep the content here as brief as possible. A link will be good enough.

Comment (e0) [reply README.md]: Have added the links now for prettier and eslint.

**Review by inghylt — APPROVED** [review]
LGTM!

**Review by mihai-sysbio — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**